src/Clean.Architecture.Core/UserAggregate/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using Clean.Architecture.SharedKernel.Interfaces;
+using Microsoft.AspNetCore.Identity;
+
+namespace Clean.Architecture.Core.UserAggregate;
+public class User : IdentityUser<int>, IAggregateRoot
```

Comment:
I don't like pulling dependency on IdentityUser and Microsoft.AspNetCore.Identity into Core project. I prefer to keep all of that in the Infrastructure project, ideally using a separate dbContext as well. See eShopOnWeb approach for reference.
https://github.com/dotnet-architecture/eShopOnWeb/tree/main/src/Infrastructure/Identity